src/main/java/com/example/tradingapp/trading/OkxOrderTracker.java

```diff
@@ -11,17 +13,23 @@
 
 @Slf4j
 @Component
+@Getter
 public class OkxOrderTracker implements OrderTracker {
 
     Map<String, Order> placedOrders = new HashMap<>();
     List<String> placedOrderIds = new ArrayList<>();
 
-    public Map<String, Order> getPlacedOrders() {
-        return placedOrders;
-    }
+    public void live(String orderId) {
+        var newStatus = OrderStatus.Live;
+
+        if (placedOrders.get(orderId) == null) {
+            log.error("Can't change status to {} for Order({}), Order already removed", newStatus, orderId);
+        } else {
+            OrderStatus oldStatus = placedOrders.get(orderId).getStatus();
+            placedOrders.get(orderId).setStatus(newStatus);
 
-    public List<String> getPlacedOrderIds() {
-        return placedOrderIds;
+            log.info("Order ({}) status change: {} -> {}", orderId, oldStatus, newStatus);
+        }
```

Comment:
I see a repeating pattern here in all three functions. Could we extract it into a separate reusable function?